plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1575,6 +1581,53 @@ SecRule REQUEST_FILENAME "@endsWith /index.php/apps/richdocuments/ajax/admin.php
     ctl:ruleRemoveTargetById=931130;ARGS:wopi_url,\
     ctl:ruleRemoveTargetById=931130;ARGS:json.wopi_url"
 
+# Enabling/disabling log reader live view
+# Administration --> Logging
+SecRule REQUEST_FILENAME "@endsWith /apps/logreader/api/settings" \
+    "id:9508634,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.1.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+# Configuring custom default app
+# Administration --> Theming --> Navigation bar settings
+SecRule REQUEST_FILENAME "@endsWith /apps/theming/ajax/updateAppMenu" \
+    "id:9508635,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.1.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+# Deleting out of office message under
+# Personal Settings --> Availability --> Absence
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9\.]+\.php/apps/dav/api/v[0-9\.]+/outOfOffice/[^/]+$" \
+    "id:9508636,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.1.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# Adjusting availability hours
+# Personal Settings --> Availability --> Availability
+SecRule REQUEST_FILENAME "@rx /remote.php/dav/calendars/[^/]+/inbox$" \
+    "id:9508637,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.1.0',\
+    ctl:ruleRemoveTargetById=932236;XML:/*,\
+    ctl:ruleRemoveTargetById=942430;XML:/*,\
+    ctl:ruleRemoveTargetById=942431;XML:/*,\
+    ctl:ruleRemoveTargetById=942432;XML:/*"
```

Comment:
Should be sufficient to use only `XML` instead of `XML:/*`.

Reply:
yup that works fine, is this a performance change?

Reply:
Probably yes as it will perform an xpath search in XML data.

Reply:
Looks like the linter didn't like that, should I revert the change?